fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -125,14 +117,6 @@ def fit(self, X, y, aux_data=None, **kwargs):
         if aux_data is None:
             raise RuntimeError("Must specify aux_data (for now)")
 
-        lagrange_multipliers = None
-        if self._KW_LAGRANGE_MULTIPLIERS in kwargs:
-            lagrange_multipliers = kwargs[self._KW_LAGRANGE_MULTIPLIERS]
-
-        number_of_lagrange_multipliers = None
-        if self._KW_NUMBER_LAGRANGE_MULTIPLIERS in kwargs:
-            number_of_lagrange_multipliers = kwargs[self._KW_NUMBER_LAGRANGE_MULTIPLIERS]
-
         # Extract the target attribute
         A = self._make_vector(aux_data, "aux_data")
```

Comment:
aux_data?

Reply:
also, we may not want to call it A anymore

Reply:
Internally, we can call it A, and there's more refactoring to come. As for renaming `aux_data` …. let's get agreement first